apps/backend/src/app/auctions/auctions.service.ts

```diff
@@ -18,16 +19,21 @@ export class AuctionsService {
     return this.auctionsRepository.save(auction);
   }
 
-  findAll() {
-    return this.auctionsRepository.find();
+  findAll(paginationQuery: PaginationQueryDto) {
+    const { limit, offset } = paginationQuery;
+
+    return this.auctionsRepository.find({
+      relations: ['category'],
+      skip: offset,
+      take: limit,
+      order: {
+        id: 'ASC',
```

Comment:
I know ID is autoincremented but wouldn't we want to use "createdAt" field to order? In case of we'll use UUID or something will happen? 😉

Reply:
Yupp, wanted to add these fields later, but ok, here you are :)